charts/astronomer/values.yaml

```diff
@@ -15,19 +15,19 @@ tolerations: []
 images:
   commander:
     repository: astronomerinc/ap-commander
-    tag: 0.18.0
+    tag: 0.21.0
```

Comment:
not ready https://app.circleci.com/pipelines/github/astronomer/commander/180/workflows/7b348c6b-ad8d-4329-b08c-07f43fa7d7f0/jobs/596

Reply:
done.